### PR TITLE
Adds Speakers

### PR DESCRIPTION
**Pull request by mdarby**
Speakers can now be CRUD'd.

* `/speakers`, et al are introduced

### CODE REVIEW

**Review by croz1007 — APPROVED** [review]

File: lib/cbus_elixir/app/app.ex [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+      iex> delete_speaker(speaker)
+      {:ok, %Speaker{}}
+
+      iex> delete_speaker(speaker)
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def delete_speaker(%Speaker{} = speaker) do
+    Repo.delete(speaker)
```
Comment (Kartstig) [inline lib/cbus_elixir/app/app.ex:89]: shouldn't this be
```elixir
Repo.delete!(speaker)
```

File: lib/cbus_elixir/app/app.ex [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      {:ok, %Speaker{}}
+
+      iex> update_speaker(speaker, %{field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def update_speaker(%Speaker{} = speaker, attrs) do
+    speaker
+    |> Speaker.changeset(attrs)
+    |> Repo.update()
```
Comment (Kartstig) [inline lib/cbus_elixir/app/app.ex:73]: !

File: lib/cbus_elixir/app/app.ex [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      {:ok, %Speaker{}}
+
+      iex> create_speaker(%{field: bad_value})
+      {:error, %Ecto.Changeset{}}
+
+  """
+  def create_speaker(attrs \\ %{}) do
+    %Speaker{}
+    |> Speaker.changeset(attrs)
+    |> Repo.insert()
```
Comment (Kartstig) [inline lib/cbus_elixir/app/app.ex:55]: !

**Review by Kartstig — CHANGES_REQUESTED** [review]
I think some of those changes can fail silently

File: lib/cbus_elixir/app/app.ex (hunk 1 above)
Comment (Kartstig) [reply lib/cbus_elixir/app/app.ex:89]: Alternatively you can do:
```elixir
{:ok, _} = Repo.delete(speaker)
```

**Comment by mdarby** [comment]
This code is auto-generated :)

**Pull request merged**